test/boost/auth_test.cc

```diff
@@ -220,25 +220,30 @@ SEASTAR_TEST_CASE(test_alter_with_timeouts) {
         // Avoid reading from memtables, which does not check timeouts due to being too fast
         e.db().invoke_on_all([] (replica::database& db) { return db.flush_all_memtables(); }).get();
 
-	    auto msg = cquery_nofail(e, "SELECT timeout FROM system_distributed.service_levels");
+        auto sl_is_v2 = e.local_client_state().get_service_level_controller().is_v2();
+	    auto msg = cquery_nofail(e, format("SELECT timeout FROM {}", sl_is_v2 ? "system.service_levels_v2" : "system_distributed.service_levels"));
         assert_that(msg).is_rows().with_rows({{
             duration_type->from_string("5ms")
         }});
 
         cquery_nofail(e, "ALTER SERVICE LEVEL sl WITH timeout = 35s");
 
-	    msg = cquery_nofail(e, "SELECT timeout FROM system_distributed.service_levels WHERE service_level = 'sl'");
+	    msg = cquery_nofail(e, format("SELECT timeout FROM {} WHERE service_level = 'sl'", sl_is_v2 ? "system.service_levels_v2" : "system_distributed.service_levels"));
         assert_that(msg).is_rows().with_rows({{
             duration_type->from_string("35s")
         }});
 
         // Setting a timeout value of 0 makes little sense, but it's great for testing
         cquery_nofail(e, "ALTER SERVICE LEVEL sl WITH timeout = 0s");
+        // we need to wait for service level controller update loop
+        // the update loop is configured to be run every 10ms in cql test env
+        sleep(10ms).get();
```

Comment:
Is there a less hacky way to ensure the update of the SL controller? Maybe there is a way to trigger the update manually?

Reply:
The way to update it manually is to create another data accessor as it was done with unit test data accessor. 
But in previous approach, unit test data accessor was overwritten by migration process (loading topology state exactly) and I wanted to avoid messing in that code only to support unit tests.

Other way (which I want to implement as a followup in a near future) is to update the state of SL controller every time the raft command is applied. 

Without those, I don't see any other way than the sleeps and it's pain in a few tests already :/

Reply:
Actually... I can do data accessor upgrade function as a method of data accessor. Then standard DA will be upgraded to raft DA and unit_test DA will be upgraded to raft_unit_test DA.